pandas/core/arrays/base.py

```diff
@@ -539,13 +540,14 @@ def shift(self, periods: int = 1, fill_value: object = None) -> ABCExtensionArra
 
         fill_value : object, optional
             The scalar value to use for newly introduced missing values.
-            The default is ``self.dtype.na_value``
+            The default is ``self.dtype.na_value``.
 
             .. versionadded:: 0.24.0
 
         Returns
         -------
-        shifted : ExtensionArray
+            ExtensionArray
+                Shifted.
```

Comment:
Can remove this line altogether

Reply:
That was intended to resolve the error: `pandas.api.extensions.ExtensionArray.shift: Return value has no description`